README.adoc

```diff
@@ -551,6 +551,32 @@ service.expects(:execute).returns(true)
 
 Choose only one to use – avoid mixing both approaches within one project.
 
+== Subclassing Test Cases
+
+Minitest uses Ruby classes, if a Minitest class inherits from another class, it will also inherit its methods 
+causing Minitest to run the parent's tests twice.
+
+[source,ruby]
+----
+# example
```

Comment:
Can you tweak it?

```suggestion
# bad
```

Reply:
I prefer not to say `bad` since it's still a valid but rare use case.

Reply:
Maybe
`# bad (unless multiple runs are the intended behavior)`

Reply:
(I find the `good`/`bad` terminology that the guides use to be not ideal, since they don't really account for nuance or context).